pytradfri/command.py

```diff
@@ -1,23 +1,28 @@
 """Command implementation."""
+from __future__ import annotations
 
 from copy import deepcopy
+from typing import Any, Optional, Callable
+
+TYPE_PROCESS_RESULT_CB = Optional[Callable[[Any], Optional[str]]]
+TYPE_ERR_CB = Optional[Callable[[str], str]]
```

Comment:
The old style annotations should be only used here. Until Python 3.10 is a default.

Reply:
Yup these do not work (yet) with the new style.